service/src/main/resources/HTMLTemplate.html

```diff
@@ -1440,9 +1447,10 @@ <h3 class="heading heading--two">Applicant</h3>
     <div class="data-list__row">
       <dt class="data-list__title">Telephone number</dt>
       <dd class="data-list__definition">
-        {% if case_data.applicant is not empty and
-        case_data.applicant.telephone is not empty %}
-          {{ case_data.applicant.telephone }}
+        {% if applicant.value.party.telephoneNumber is not empty %}
+        {% if applicant.value.party.telephoneNumber.telephoneNumber is not empty %}
```

Comment:
Correct indentation. It makes reading code easier / less of a pain.

Reply:
👍